src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersLowBalanceAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { WarningFilled } from '@edx/paragon/icons';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import { LOW_BALANCE_CONTACT_ADMIN_TEXT, LOW_BALANCE_ALERT_HEADING, LOW_BALANCE_ALERT_TEXT } from './data/constants';
+
+const EnterpriseOffersLowBalanceAlert = () => {
+  const {
+    enterpriseConfig: { adminUsers },
+  } = useContext(AppContext);
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const actions = hasAdminEmails ? [
+    <MailtoLink
```

Comment:
nit: Technically, I think the mock has the "Contact administrator" as a tertiary button style, but in some quick hacking around, don't think there's an easy way to get the styles we want with combining `MailtoLink` and `Button` together (e.g., `<Button as={MailtoLink}>`). 

Let's keep as is :)